metadata/overlays/istio/virtual-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ui
```

Comment:
Can we rename this `metadata-ui` so it's consistent with some of the other VirtualServices (ie. `ml-pipeline-ui`, `katib-ui`, etc.)?

Reply:
Yes, I would actually prefer to do as you suggested, but I wasn't sure if I should do to the presence of a name-prefix in the kustimzation.yaml in`base`
(https://kubectl.docs.kubernetes.io/pages/reference/kustomize.html#nameprefix)
Perhaps it'd be better to just do away with that as well since it makes things less clear when skimming the files

Reply:
I may not have done it properly when I added it to my cluster. I just ran `kustomize build overlays/istio/ | kubectl apply -f -` and it showed up as `ui`. Perhaps when it's actually run with `kfctl` it respects the prefix set in `base`.

*full-disclosure I'm shooting from the hip on this stuff just trying to get things to work!*